docs/basic-docs/atomicdex/atomicdex-desktop/add-coin-to-atomicdex-desktop.md

```diff
@@ -0,0 +1,260 @@
+
+This involves:
+- Setting up two operational [electrum servers](https://electrumx.readthedocs.io/en/latest/)
+- Setting up a [block explorer](https://github.com/gcharang/komodo-install-explorer)
+- Performing a [test swap](https://github.com/KomodoPlatform/coins/tree/master/swaps)
+- Submitting a Pull Request (PR) to the [coins repository](https://github.com/KomodoPlatform/coins)
+
+Once these steps are complete, you can submit a PR to the [AtomicDEX-Desktop repository](https://github.com/KomodoPlatform/atomicDEX-Desktop)
+
+Here is an [actual example](https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/1504) of the files that need to be modified:
+1) Add information to [0.x.x-coins.json](https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/dev/assets/config/0.5.4-coins.json). Note: This filename changes depending on the current version of the Desktop app.
+2) Add [TradingView IDs](https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/dev/atomic_defi_design/Dex/Constants/General.qml#L487) (optional) if available. This is used to show price graphs.
+3) Add an entry to the [Style.qml file](https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/dev/atomic_defi_design/Dex/Constants/Style.qml#L278), to define the text color used to display the coin name. Be sure to check that this color works well and is readable within both lite and dark themes.
+4) Add a coin icon to the [atomic_defi_design/assets/images/](https://github.com/KomodoPlatform/atomicDEX-Desktop/tree/dev/atomic_defi_design/assets/images/coins) folder. This should be in PNG format, optimised for filesize (less than 500kb), and at least 128 x 128 pixels in size.
+5) Add a reference to the icon image file into [qml.qrc](https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/dev/atomic_defi_design/qml.qrc#L162)
+
+## Update coins.json 
+
+| parameter              | Type     | Description           |
+| ---------------------- | -------- | --------------------- |
+| coin                   | string   | Ticker of the coin. Must match the "coin" field for an entry in the [coins repository](https://github.com/KomodoPlatform/coins/blob/master/coins#L5138) If the coin is on more than one network, use COIN-NETWORK as ticker, eg BTC-BEP20. |
+| name                   | string   | Full name of the coin. Should match the "fname" field for an entry in the [coins repository](https://github.com/KomodoPlatform/coins/blob/master/coins#L5138). Coins on more than one network share the same name. |
+| type                   | string   | Defines the category or network the coin is from. For example, `ERC20` for tokens on the Ethereum network; `Smart Chain` for Komodo Platform Antara smart chains; `UTXO` for Bitcoin forks (LTC, DOGE etc) |
+| asset                  | string   | (required for Smart Chains) Should match the "asset" value in the [coins repository](https://github.com/KomodoPlatform/coins/blob/master/coins#L7342) |
```

Comment:
this is not needed in Desktop config, only https://github.com/cipig/coins/blob/master/coins has this field
ADEX Desktop only knows the tickers, like in TKL
```
  "TKL": {
    "coin": "TKL".
    "type": "Smart Chain",
```
and `"type": "Smart Chain"` identifies this entry as a KMD smart chain

Reply:
@Milerius can you please confirm which of these fields are not required in https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/dev/assets/config/0.5.4-coins.json#L4988-L4991 ?

I can's see `type` in https://github.com/KomodoPlatform/coins/blob/master/coins#L6653-L6665 so assume that is needed. 

`asset` only appears in smart chains for RICK/MORTY, so I assume this is not needed.

`name` appears to correspond with `fname` in coins repo. 

`coin` seems like unneeded duplication of the object key (though it links to `coin` in coins repo)

Reply:
With local testing, removing `asset` is safe, but removing any of the other fields results in a crash on login.